advanced_data_analyst_bootcamp.model.lkml

```diff
@@ -25,23 +39,13 @@ explore: order_items {
     relationship: many_to_one
   }
 
-  join: products {
-    type: left_outer
-    sql_on: ${inventory_items.product_id} = ${products.id} ;;
-    relationship: many_to_one
-  }
 
-  join: distribution_centers {
-    type: left_outer
-    sql_on: ${products.distribution_center_id} = ${distribution_centers.id} ;;
-    relationship: many_to_one
-  }
 }
 
 explore: events {
-  join: event_session_facts {
+  join:  session_event_facts {
     type: left_outer
-    sql_on: ${events.session_id} = ${event_session_facts.session_id} ;;
+    sql_on: ${events.session_id} = ${ session_event_facts.session_id} ;;
```

Comment:
This, when invoked, will create a SQL error.  Be sure to test your changes to make sure they work!  That space between the { and session_event_facts prevents Looker from seeing that as a substitution operator and instead it reads it as raw SQL even though it is purple.  The resulting SQL is:

FROM public.events  AS events
LEFT JOIN session_event_facts ON events.session_id = ${ session_event_facts.session_id}